lib/bloc/user_profile/breez_user_model.dart

```diff
@@ -9,12 +10,14 @@ class BreezUserModel {
   String color = '';
   String animal = '';
   String _image;
+  SecurityModel securityModel;
```

Comment:
securityModel and waitingForPin should be final. Also all other model variables should be final as models are immutable.